tests/rptest/clients/kafka_cli_tools.py

```diff
@@ -247,6 +247,83 @@ def get_api_versions(self):
         return self._run("kafka-run-class.sh", [],
                          "kafka.admin.BrokerApiVersionsCommand")
 
+    def describe_producers(self, topic, partition):
+        expected_columns = ["ProducerId", "ProducerEpoch", "LastSequence", "LastTimestamp", "CoordinatorEpoch", "CurrentTxnStartOffset"]
+        self._redpanda.logger.debug(
+            "Describe producers for topic %s partition %s", topic, partition)
+        cmd = [self._script("kafka-transactions.sh")]
+        cmd += ["--bootstrap-server", self._redpanda.brokers()]
+        cmd += ["describe-producers"]
+        cmd += ["--topic", topic]
+        cmd += ["--partition", str(partition)]
+        res = self._execute(cmd)
+
+        producers = []
+        info_str = res.split("\n")[0]
+        info_key = info_str.strip().split("\t")
+        assert len(info_key) == len(expected_columns)
+
+        lines = res.split("\n")[1:-1]
```

Comment:
why the last line is ignored (`[0,1,2,3][1:-1]==[1,2]`)?

Reply:
first is header 
last is empty string after split

Reply:
add an assert checking it's empty